internal/v5/api.go

```diff
@@ -994,6 +1009,7 @@ func checkExtraInfoKey(key string, field string) error {
 // GET id/meta/perm
 // https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetaperm
 func (h *ReqHandler) metaPerm(entity *mongodoc.BaseEntity, id *router.ResolvedURL, path string, flags url.Values, req *http.Request) (interface{}, error) {
```

Comment:
If there's no wordpress in a stable channel, then ~charmers/wordpress won't resolve.
But after PR about to be proposed, PUT /v5/~charmers/wordpress-0/meta/perm?channel=stable
will work (ignoring the channel attribute) and in the PR after that one, the meta/perm endpoint will inspect the channel parameter to decide which perms to use, so that PUT request will update the stable ACLs even if there is no wordpress in a stable channel.